app/src/findexecutable.js

```diff
@@ -0,0 +1,39 @@
+// Forked from https://github.com/springernature/hasbin
+// Because it has been archived.
+// MIT License
```

Comment:
The original license of `hasbin` is MIT, maybe I should relicense to GPL here to fit vdhcoapp.

Reply:
Already rewritten completely.